src/lightkurve/correctors/crowdingcorrector.py

```diff
@@ -101,9 +102,12 @@ def correct(self, crowdsap=None, flfrcsap=None):
         
         #Calculate the new uncertainties
         flux_err_corr = self.flux_err.value/FLFRCSAP
+
+        #Define units for the lc
+        unit = u.electron/ u.s
```

Comment:
- It'd be safer not to assume unit to be `e/s`. E.g., use `self.lc.flux.unit` instead.
- corrected `lc` does not have not-flux columns (e.g., `cadenceno`, `quality`, etc.) and metadata of the original lc. It could be surprising to users. I suggest `corrected_lc` be created by first creating a copy of the original `lc.copy()`. Then its `flux`, `flux_err` can be replaced with the corrected ones.

Reply:
I love these suggestions and have adjusted accordingly. Thanks